torchao/dtypes/fbgemm_fp8_tensor.py

```diff
@@ -82,6 +90,53 @@ def to(self, *args, **kwargs):
             self.float8_data.to(device),
             self.scale.to(device),
             self.activation_scale_ub.to(device),
+            self.data_to_scale_dim,
+            self.dtype,
+        )
+
+    def _transpose_and_reshape(self):
+        """This is added for resharding support, since the resharding logic for the model we are
```

Comment:
Do these these next two functions  need to be methods or can they be implementations of the actual ops

Reply:
these should be methods, it's specific for the hack we are doing